### PR TITLE
Allow any command for runcmd= option

### PR DESCRIPTION
**Pull request by gurevichmark**
#5446 

Allow any command specification for `runcmd=` option.
Also, modify usage and man page to show any command can be used.

### CODE REVIEW

File: docs/source/guides/admin-guides/references/man8/nodeset.8.rst [hunk 1]
```diff
@@ -113,10 +113,9 @@ A user can supply their own scripts to be run on the mn or on the service node (
  
 
 
-\ **runcmd=bmcsetup**\ 
+\ **runcmd=**\ \ *command*\ 
  
- This instructs the node to boot to the xCAT nbfs environment and proceed to configure BMC
- for basic remote access.  This causes the IP, netmask, gateway, username, and password to be programmed according to the configuration table.
+ This instructs the node to boot to the xCAT genesis environment and specified command to be executed.  This causes the IP, netmask, gateway, username, and password to be programmed according to the configuration in node object definition.
```
Comment (zet809) [inline docs/source/guides/admin-guides/references/man8/nodeset.8.rst]: I suggest to put the line `This cause the IP....` for the example 3 below. `runcmd=bmcsetup`

File: docs/source/guides/admin-guides/references/man8/rinstall.8.rst [hunk 2]
```diff
@@ -70,9 +70,9 @@ If \ **-c**\  is specified, it will then run \ **rcons**\  on the node. This is
  
 
 
-\ **runcmd=bmcsetup**\ 
+\ **runcmd=**\ \ *command*\ 
  
- This instructs the node to boot to the xCAT nbfs environment and proceed to configure BMC for basic remote access.  This causes the IP, netmask, gateway, username, and password to be programmed according to the configuration table.
+ This instructs the node to boot to the xCAT genesis environment and specified command to be executed.  This causes the IP, netmask, gateway, username, and password to be programmed according to the configuration in node object definition.
```
Comment (zet809) [inline docs/source/guides/admin-guides/references/man8/rinstall.8.rst]: Same comments that put the line `This cause...` as the explanation  of `runcmd=bmcsetup`

File: xCAT-client/pods/man8/nodeset.8.pod [hunk 3]
```diff
@@ -78,10 +78,9 @@ If you would like to run a task after deployment, you can define that task with
 
 Display the current boot loader config file description for the nodes requested. When B<disjointdhcps> is set, using B<-a> to display them on all available service nodes.
 
-=item B<runcmd=bmcsetup>
+=item B<runcmd=>I<command>
 
-This instructs the node to boot to the xCAT nbfs environment and proceed to configure BMC
-for basic remote access.  This causes the IP, netmask, gateway, username, and password to be programmed according to the configuration table.
+This instructs the node to boot to the xCAT genesis environment and specified command to be executed.  This causes the IP, netmask, gateway, username, and password to be programmed according to the configuration in node object definition.
```
Comment (zet809) [inline xCAT-client/pods/man8/nodeset.8.pod]: Same comments

File: xCAT-client/pods/man8/rinstall.8.pod [hunk 4]
```diff
@@ -40,9 +40,9 @@ Skip the kernel version checking when injecting drivers from osimage.driverupdat
 
 If you would like to run a task after deployment, you can define that task with this attribute.    
 
-=item B<runcmd=bmcsetup>
+=item B<runcmd=>I<command>
 
-This instructs the node to boot to the xCAT nbfs environment and proceed to configure BMC for basic remote access.  This causes the IP, netmask, gateway, username, and password to be programmed according to the configuration table.
+This instructs the node to boot to the xCAT genesis environment and specified command to be executed.  This causes the IP, netmask, gateway, username, and password to be programmed according to the configuration in node object definition.
```
Comment (zet809) [inline xCAT-client/pods/man8/rinstall.8.pod]: Same comments

**Comment by gurevichmark** [comment]
@zet809 Updated man pages to move `This causes the IP....` from option description to the example description.

File: docs/source/guides/admin-guides/references/man8/nodeset.8.rst [hunk 5]
```diff
@@ -199,6 +198,16 @@ root directory and the TFTP xCAT  subdirectory.   /tftpboot  and
  
 
 
+3. Boot node1 into genesis environment and execute bmcsetup script. This causes the IP, netmask, gateway, username, and password to be programmed according to the configuration in node object definition.
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/references/man8/nodeset.8.rst]: Probably be consistent and call `xCAT genesis environment` whenever we reference genesis.

Comment (gurevichmark) [reply docs/source/guides/admin-guides/references/man8/nodeset.8.rst]: Updated commit to make text consistent.

**Comment by zet809** [comment]
Looks good to me, I agree to merge

**Pull request merged**